lua-docs/content/reference/Map.md

```diff
@@ -0,0 +1,39 @@
+---
+description: Particubes
+keywords: particubes, game, mobile, scripting, cube, voxel, world
+---
+
+# Map
+
+`Map` is a global variable that represents the world map.
+
+## Fields
+
+### Width (Integer, read-only)
+
+Returns the map's width, measured in cubes.
+
+### Height (Integer, read-only)
+
+Returns the map's height, measured in cubes.
+
+### Depth (Integer, read-only)
+
+Returns the map's depth, measured in cubes.
+
+### Scale (Float, read-only)
+
+Returns the map's scale factor, measured in pixels per cube.
+
+### Set (Function, read-only)
+
+Function to define the world map.
+
+Parameters:
+
+- map name (string)
+- scale factor (float)
```

Comment:
Is scale factor optional? If yes, we should indicate the default value. (8 I think)

Reply:
As of today, it is not optional.

Reply:
👍